feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go

```diff
@@ -53,28 +68,61 @@ func TestMain(m *testing.M) {
 //    https://github.com/fullstorydev/grpcurl
 //
 
+// gNOIKillProcess kills a daemon on the DUT, given its name and pid.
+func gNOIKillProcess(ctx context.Context, t *testing.T, dut *ondatra.DUTDevice, pName string, pID uint32) {
+	gnoiClient := dut.RawAPIs().GNOI().Default(t)
+	killRequest := &gnps.KillProcessRequest{Name: pName, Pid: pID, Signal: gnps.KillProcessRequest_SIGNAL_TERM,
+		Restart: true}
+	killResponse, err := gnoiClient.System().KillProcess(ctx, killRequest)
+	t.Logf("Got kill process response: %v\n\n", killResponse)
+	if err != nil {
+		t.Fatalf("Failed to execute gNOI Kill Process, error received: %v", err)
+	}
+}
+
+// findProcessByName uses telemetry to find out the PID of a process
```

Comment:
need a period at the end of the sentence. And it should say that it returns 0 if there is no such process.

Reply:
Resolved in latest commit